src/constants/DefaultFields.ts

```diff
@@ -20,7 +20,21 @@ export const DefaultFields: Record<string, DefaultField> = Object.freeze({
   CURRENT_CLASS: {
     id: DefaultFieldKey.CURRENT_CLASS,
     is_default: true,
-    name: "Current Class",
+    name: "Class",
+    question_type: QuestionTypes.MULTIPLE_CHOICE,
+    options: [],
+  },
+  CURRENT_SESSION: {
+    id: DefaultFieldKey.CURRENT_SESSION,
+    is_default: true,
+    name: "Session",
+    question_type: QuestionTypes.MULTIPLE_CHOICE,
+    options: [],
+  },
+  CURRENT_PREFERRED_CLASS: {
+    id: DefaultFieldKey.CURRENT_PREFERRED_CLASS,
+    is_default: true,
+    name: "Preferrences",
     question_type: QuestionTypes.MULTIPLE_CHOICE,
     options: [],
   },
```

Comment:
These two aren't actually part of the default view of the main registration table, so I think I'll move them to another object to make naming more consistent with the design.

Reply:
are you moving them in this PR? I don't think those two fields are being used anywhere outside the sidebar, so feel free to keep those within the necessary components